### PR TITLE
Comment share

### PR DESCRIPTION
**Pull request by Luuwa**
WIP PR for #104

### CODE REVIEW

**Comment by Luuwa** [comment]
[screen-capture (1).webm](https://user-images.githubusercontent.com/22305845/204130358-232461bf-9524-489c-951d-51c17e33630d.webm)

**Comment by Luuwa** [comment]
After visiting the url we created, subcomments of all comments are opened. because of this its url cannot center the shared comment
[PR#104bug.webm](https://user-images.githubusercontent.com/22305845/204130934-0ab997d6-3876-47bc-bc1b-a1486cee2ff6.webm)

**Comment by Luuwa** [comment]
[PR104bug.webm](https://user-images.githubusercontent.com/22305845/204131004-9df2e083-3777-4267-ab2d-f4453001ec61.webm)

**Comment by cansirin** [comment]
> After visiting the url we created, subcomments of all comments are opened. because of this its url cannot center the shared comment.

`expanded` state is getting assigned to CommentItem later than when browser is trying to reach to the comment where 
`hash === id`. All comments should be expanded in order to browser go to that comment anyway, so I think we should find a better way to handle this.

**Comment by cansirin** [comment]
Can not logged in users see  `MoreOptionsDropdown` for comments?

**Comment by Luuwa** [comment]
> Can not logged in users see `MoreOptionsDropdown` for comments?

They can see

**Comment by Luuwa** [comment]
focus

File: apps/pano/src/features/comment/MoreOptionsDropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        open={openToast}
+        setOpen={setOpenToast}
+        duration={5000}
+        title="Link kopyalandı"
+      />
+    </>
+  );
+};
+
+const SocialMediaShareButtons = ({ commentUrl }: { commentUrl: string }) => {
```
Comment (usirin) [inline apps/pano/src/features/comment/MoreOptionsDropdown.tsx:72]: It feels like we need to make this a generic component, but we can do that later in a follow up PR.

**Review by usirin — APPROVED** [review]

**Pull request merged**